### ISSUE TITLE
Ampersand selector not working properly

### ISSUE THREAD

**Issue by riyazshaikh** [Squarespace/less-compiler#1]
```
.collection-layout-default {
    #header {
        .header-palette-banner& {
            /* styles */
        }
    }
}
```

​
Currently outputs (v1.3.3 on Squarespace)
`.header-palette-banner .collection-layout-default #header{ /*styles */ }`
​

Should be
`.header-palette-banner.collection-layout-default #header{ /*styles */ }`
​

**Comment by phensley** [comment]
Thanks @riyazshaikh I'll take a look.

**Comment by phensley** [comment]
I have a fix for this in code review, should be merged today.

**Comment by riyazshaikh** [comment]
Awesome! But is this on production branch (v1.3.3) or your own feature branch (v2.4)?

**Comment by phensley** [comment]
Code is on the internal repository for review, then I'll push it here.  This is a fix for the `1.x` branch (1.3.3).  The bug doesn't exist on `master` (2.4.0)

**Comment by phensley** [comment]
Hey @riyazshaikh I've released `1.0.18` which includes the fix to the ampersand bug.  This should go out with the next deploy, probably in a few hours.  I'll update when it hits production.

**Comment by riyazshaikh** [comment]
Verified in production. Thanks a lot!
